tasks/make_zipapp.py

```diff
@@ -180,7 +180,7 @@ def get_dependencies(whl, version):
             platforms = []
             platform_positions = WheelDownloader._marker_at(markers, "sys_platform")
             deleted = 0
-            for pos in platform_positions:  # can only be ore meaningfully
+            for pos in platform_positions:  # can only be more meaningfully
```

Comment:
"more" doesn't make sense here. Was the typo meant to be "one"? I'm not sure if that makes any more sense, but at least it's a grammatically complete phrase...

Reply:
tbh - that whole comment didn't make sense to me either - but pre-commit wanted to change it. I'm happy to leave it out, since it's in a file I was not really interested in :)